platform/kindle/koreader.sh

```diff
@@ -232,20 +236,20 @@ if [ "${STOP_FRAMEWORK}" == "no" -a "${INIT_TYPE}" == "upstart" ] ; then
 fi
 
 # stop cvm (sysv & framework up only)
-if [ "${STOP_FRAMEWORK}" == "no" -a "${INIT_TYPE}" == "sysv" ] ; then
+if [ "${STOP_FRAMEWORK}" = "no" ] && [ "${INIT_TYPE}" = "sysv" ] ; then
 	logmsg "Stopping cvm . . ."
 	killall -stop cvm
 fi
 
 # finally call reader
 logmsg "Starting KOReader . . ."
 # That's not necessary when using KPVBooklet ;).
-if [ "${FROM_KUAL}" == "yes" ] ; then
+if [ "${FROM_KUAL}" = "yes" ] ; then
 	eips_print_bottom_centered "Starting KOReader . . ." 1
 fi
 
 # we keep maximum 500K worth of crash log
-cat crash.log 2> /dev/null | tail -c 500000 > crash.log.new
+< crash.log tail -c 500000 > crash.log.new
```

Comment:
Yes, I think that's much clearer and therefore better. Besides, as you pointed out the next line otherwise errors as well.

```
$ mv -f crash.log.new crash.log
mv: cannot stat 'crash.log.new': No such file or directory
```